Change release to coredns v1.6.9.

ericgraf opened the pull request:
Change coredns to version v1.6.9 and revert a tag change that is done in v1.7.0

Release notes: [https://coredns.io/2020/03/24/coredns-1.6.9-release/](https://coredns.io/2020/03/24/coredns-1.6.9-release/)

stone-z commented on helm/coredns-app/values.yaml:
```diff
@@ -2,7 +2,7 @@
 # Declare variables to be passed into your templates.
 
-name: coredns
+name: coredns-app
```
Is this still part of reverting the previous update? This should be independent of the `coredns` version

ericgraf replied:
I think was was left over from my last testing. Thanks for pointing this out. That should not be there I'll fix it.

stone-z reviewed (approved):
This seems ok to me

ericgraf commented:
Here is a aggregated summary of the Noteworthy Changes that this change pulls in from coredns

## v1.6.8 = v1.6.9
plugin/azure: Add private DNS support for azure plugin 
plugin/cache: Fix negative cache masking cases 
plugin/cache: explain drop metric 
plugin/forward: Add configuration flag to set if RecursionDesired should be set on health checks 
plugin/forward: Add exponential backoff to healthcheck 
plugin/forward: Add max_concurrent option 
plugin/hosts: Modifies NODATA handling 
plugin/kubernetes: fix metadata 
plugin/kubernetes: Return all records with matching IP for reverse queries 
plugin/metrics: Add query type to latency as well
plugin/pkg/up: Make default intervals shorter

## v1.6.7
Add mips64le to released architectures 
Fix HostPortOrFile to support IPv6 addresses with zone 
plugin/acl: Document metrics in README 
plugin/cache: Registry cache_miss logic 
plugin/cache: Update comment to conform to the implementation 
plugin/{forward, grpc}: Dedup policy implement between grpc and proxy plugin 
plugin/kubernetes: Bump kubernetes plugin schema version 
plugin/{kubernetes, etc}: Resolve TXT records via CNAME 
plugin/logs: Docs: update README and log plugin 
plugin/sign: Add expiration jitter 

## v1.6.6
plugin/bufsize: A new bufsize plugin to prevent IP fragmentation and DNS Flag Day 2020 
plugin/transfer: Fixed calling wg.Add in main goroutine to avoid race conditions 
plugin/pprof: Fixed a reloading issue 
plugin/health: Fixed a reloading issue 
plugin/redy: Fixed a reloading issue 
plugin/cache: Added a serve_stale option similar to unbound’s serve_expired 
plugin/sign: Fix signing of authoritative data 
pkg/reuseport: Move the core server listening functions to a new package

stone-z commented:
Tested on AWS on ginger, passed CNCF conformance
